lib/src/Pos/Util/Servant.hs

```diff
@@ -717,29 +720,56 @@ instance ReportDecodeError api =>
 -- Custom query flag
 -------------------------------------------------------------------------
 
+-- This type is used as a helper to implement custom query flags.
+-- Instead of using `QueryFlag "some_flag"` which should serialize
+-- into boolean flag now we can say `CustomQueryFlag "some_flag" SomeFlag`
+-- where SomeFlag has instance of Flaggable. This way we won't be using
+-- Boolean type for all flags but we can implement custom type.
 data CustomQueryFlag (sym :: Symbol) flag
 
+-- TODO (akegalj): add roundtrip test
```

Comment:
@KtorZ where do you think I should I add this roundtrip test. I see Test.Pos.Util.Tripping module defines `trippingBuildable` which seems like a good candidate to build this test - but not sure in which module to put it?

Reply:
What about simply adding tests in `lib/test/Test/Pos/Util/ServantSpec.hs` as they would probably be if they existed before ^.^ ?

Reply:
hm - then I would have to add `cardano-wallet-new` dependency to `cardano-test` test suite (as I would have to test `ForceNtpCheck` roundtrip which lives in wallet-new). I wasn't awere we are grupping tests in `lib/` but I see other libs are imported as well (previously we had thests within `wallet-new/...` path)

:+1: got it now, thanks!